recipes/LibriSpeech/librispeech_prepare.py

```diff
@@ -296,30 +325,23 @@ def create_csv(
     csv_lines = [["ID", "duration", "wav", "spk_id", "wrd"]]
 
     snt_cnt = 0
+    line_processor = functools.partial(process_line, text_dict=text_dict)
     # Processing all the wav files in wav_lst
-    for wav_file in tzip(wav_lst):
-        wav_file = wav_file[0]
-
-        snt_id = wav_file.split("/")[-1].replace(".flac", "")
-        spk_id = "-".join(snt_id.split("-")[0:2])
-        wrds = text_dict[snt_id]
-
-        signal, fs = torchaudio.load(wav_file)
-        signal = signal.squeeze(0)
-        duration = signal.shape[0] / SAMPLERATE
-
+    for row in parallel_map(line_processor, wav_lst, chunk_size=8192):
```

Comment:
why you hardcoded the `chunk_size` value while in the CommonVoice recipe this is not the case?

Reply:
I found out that `torchaudio.info` is already very fast for these FLAC files (to the point parallelism is basically just extra). Bringing the chunk size up decreases overhead and increases worker CPU usage (thus effective utilization) in my testing.

For CommonVoice, I'm pretty sure workers basically already maxxed out all allocated cores, so it wasn't a problem there. We could increase chunk size without many drawbacks other than that the progress bar would update way more slowly but I don't think it'd really help in practice.

Reply:
Ok ty. Could you please add just a small comment explaining that?